server/webapp/WEB-INF/rails/app/controllers/admin/pipelines_controller.rb

```diff
@@ -286,6 +286,10 @@ def load_group_list
         group_array.push({:group => group})
         group_list.push(group)
       end
+
+      group_array = group_array.sort_by { |group| group[:group].downcase }
+      group_list = group_list.sort_by { |group| group.downcase }
```

Comment:
The ruby way would be to do: `group_list.sort_by(&:downcase)`, but what you have works too.

Reply:
makes sense, will update it

Reply:
updated